### ISSUE TITLE
kernel modules loader

### ISSUE THREAD

**Issue by ivg** [BinaryAnalysisPlatform/bap#633]
The LLVM-3.4 based loader just segfaults. I would still consider that this is a bug on our side, as the `llvm-objdump-3.4` tool is capable of dumping the information without any segfaults. 

The ida based loader also doesn't work as expected, for example, for `msdos.ko` we have only one function with few lines of assembly. IDA Pro itself produces a much bigger disassembly. (As well as llvm-objdump and objdump). It looks like that flags are not propagated correctly...

To summarize:

- [ ] llvm-loader should work
- [ ] ida loader should also work

**Comment by ivg** [comment]
fixed in #654 and #686
